pdfly/cli.py

```diff
@@ -227,8 +227,7 @@ def compress(
 ) -> None:
     pdfly.compress.main(pdf, output)
 
-
-@entry_point.command(name="x2pdf", help=pdfly.x2pdf.__doc__)  # type: ignore[misc]
+@entry_point.command()
```

Comment:
Why removing the `name` & `help` strings?

Reply:
Oh I miss clicked 😞